### ISSUE TITLE
get 100 random EMDB entries for the training set, and 10 for the test set (10A/px, make sure they’re not all ribosomes or otherwise related).

### ISSUE THREAD

**Issue by alisterburt** [warpem/miss-alignment#4]

**Comment by McHaillet** [comment]
I was expecting many ribosomes in the subset that I now generated, but, interestingly, instead it turned out to be a lot of Sars-cov2 structures 😛 

```
Random EMDB subset contains 4 structures with ribo in the title.
Random EMDB subset contains 10 structures with sars in the title.
```

**Comment by alisterburt** [comment]
😂😂😂

**Comment by McHaillet** [comment]
Note: I generated a new dataset with random number seed 43. One map didn't downloaded (EMD-23680), so I manually added EMD-37149.

**Comment by alisterburt** [comment]
seems like this is now done!
